create_infotexts.py

```diff
@@ -461,6 +493,16 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
 
 
         self.data["depicted_place"] = place_as_wikitext
+        
+    
+    def process_keywords(self, item, mapping):
```

Comment:
Ah yes, I figured out the logic directly in L237-L242 and forgot to add it as a object method. Done!